common.blocks/menu/menu.bemhtml

```diff
@@ -38,13 +38,23 @@ block('menu')(
             _checkedItems : checkedItems,
             _menuMods : {
                 theme : mods.theme,
-                disabled : mods.disabled
+                disabled : mods.disabled,
+                mode : mods.mode
             }
         });
     }),
     attrs()(function() {
-        var attrs = { role : 'menu' };
-        this.mods.disabled || (attrs.tabindex = 0);
+        var ctx = this.ctx,
+            mods = this.mods,
+            attrs = {
+                role : 'menu',
+                'aria-disabled' : mods.disabled
```

Comment:
We don't support such value - see [documentation](https://en.bem.info/libs/bem-components/v2.2.1/desktop/menu/#disabled)